Makefile

```diff
@@ -31,8 +36,24 @@ endif
 
 COVER_THRESHOLD := $(shell grep '^name: cover' .github/workflows/ci-go-cover.yml | cut -c13-)
 
+
+define MOCK_template
```

Comment:
add the generated mock to `CLEANFILES` here and define a new target `clean: ; $(RM) $(CLEANFILES)`

Reply:
Done!